testsuite/cli/src/client_proxy.rs

```diff
@@ -210,9 +211,21 @@ impl ClientProxy {
             sync_on_wallet_recovery,
             temp_files: vec![],
             quiet_wait,
+            url,
         })
     }
 
+    /// Gets a websocket client for the same node `DiemClient` connects to
+    pub async fn websocket_client(&self) -> Result<WebsocketClient, diem_client::Error> {
```

Comment:
why we add it here? this is confusing, why cli need this, you may add when cli has subscribe to event command created

Reply:
because the client_proxy is what's used for the smoke tests. I'll add a separate helper function for it